packages/core/src/schema/models/commits.ts

```diff
@@ -9,31 +9,23 @@ import {
   varchar,
 } from 'drizzle-orm/pg-core'
 
-import { latitudeSchema, PromptSnapshot, promptSnapshots } from '..'
+import { DocumentSnapshot, documentSnapshots, latitudeSchema } from '..'
 import { timestamps } from '../schemaHelpers'
-import { users } from './users'
-import { workspaces } from './workspaces'
 
 export const commits = latitudeSchema.table(
   'commits',
   {
-    id: bigserial('id', { mode: 'bigint' }).notNull().primaryKey(),
+    id: bigserial('id', { mode: 'number' }).notNull().primaryKey(),
     uuid: uuid('uuid')
       .notNull()
       .unique()
       .default(sql`gen_random_uuid()`),
-    nextCommitId: bigint('next_commit_id', { mode: 'bigint' }).references(
+    nextCommitId: bigint('next_commit_id', { mode: 'number' }).references(
       (): AnyPgColumn => commits.id,
       { onDelete: 'restrict' },
     ),
-    title: varchar('title', { length: 256 }).notNull(),
```

Comment:
Why? when title is not required?

Reply:
i haven't seen the need for commit title or descriptions in the UI yet

Reply:
<img width="424" alt="image" src="https://github.com/user-attachments/assets/25cfdb93-77ed-4bf1-b7ac-04a570c380d5">

Reply:
Or do we want to allow spaces in the name?

Reply:
i don't see any commit name/description in the picture? aren't you confusing commit name with document name?

Reply:
Upps sorry that's right I was thinking on prompts. But following the analogy of git commits wouldn't make sense to write at least a title when the changes are commited?

Reply:
I don't think so analogies break at some point